orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/aws/lambda/lambdaFunctionTask.java

```diff
@@ -44,8 +51,14 @@ public TaskResult execute(@Nonnull Stage stage) {
 
     Map<String, Object> task = new HashMap<>(stage.getContext());
 
+    String operationName = (String) task.get("operation");
```

Comment:
Are there other required parameters for `lambdaFunctionTask` to succeed? Seems likely from a quick look at the related clouddriver code. It may be better to have a private pojo class in here with constraint annotations (like `@NotNull`), then use `stage.mapTo()` and check for validation errors on the resulting object.

Reply:
I like the idea of Orca being lean on validation and type definitions of clouddriver operations unless it's critical to performing orchestration logic. Clouddriver is the authority of the payload structure, and so should also be the authority on acceptable values.